src/util/cmdline.cpp

```diff
@@ -121,6 +123,14 @@ bool cmdlinet::parse(
     args = vm["input-file"].as<std::vector<std::string>>();
     verification_file = args.back();
   }
+  if (vm.count("dummymain")) // dummy main for Solidity frontend
+  {
+    sol_main = vm["dummymain"].as<std::vector<std::string>>().back();
```

Comment:
Could you explain why "--function" is not enough?

Reply:
The new Solidity frontend relies on clang_c_frontend to convert ESBMC intrinsics (Line 34 and Line 77 in `src/solidity-frontend/solidity_language.cpp`):

```cpp
34   clang_c_module->parse(sol_main_path, msg);
...
77   clang_c_module->convert_intrinsics(new_context, msg);
```

If only '--function' is provided but 'dummymain' is not provided, this conversion will fail with the error message below: 
```cpp
error: no input files
Assertion failed: (Jobs.size() == 1), function buildASTs, file ../src/clang-c-frontend/AST/build_ast.cpp, line 90.
```

'dummymain' was added as a workaround to resolve this error.